### ISSUE TITLE
Question regarding Joint-FT

### ISSUE THREAD

**Issue by Divadi** [gqk/LAE#3]
Hello Authors,

Thank you for releasing the code. Could I ask how you implemented the Joint-FT experiment? More specifically:

- Which codebase/configs was used?
- What training settings were used for this experment? (Learning rate, epochs, learning rate schedule, etc).

Best,
Jinhyung Park

**Comment by gqk** [comment]
Hi Jinhyung,

Thank you for your interest in our work! As mentioned in **Sec. 5.2**, we trained Joint-FT and Seq-FT using the recommended fine-tuning strategy of ViT [5], with the same number of training epochs as ours.

_[5] Alexey Dosovitskiy, Lucas Beyer, Alexander Kolesnikov, Dirk Weissenborn, Xiaohua Zhai, Thomas Unterthiner, Mostafa Dehghani, Matthias Minderer, Georg Heigold, Syl- vain Gelly, Jakob Uszkoreit, and Neil Houlsby. An image is worth 16x16 words: Transformers for image recognition at scale. In Proceedings of the International Conference on Learning Representations (ICLR), 2021._

  - Which codebase/configs were used?

    We used this codebase with `data.num_init_classes=0` and `data.num_increment_classes=N`, where N is 100 or 200 for CIFAR100/ImageNet-R.

  - What training settings were used for this experiment? (Learning rate, epochs, learning rate schedule, etc.)

    Learning Rate/Batch Size: 0.005/64 (It should be 0.01/128, but it requires more GPU memory)
    Epochs: 5/50 for CIFAR100/ImageNet-R
    Optimizer/LR Schedule: SGD/Cosine

  I hope this addresses your questions. If you have any further inquiries or need additional information, please feel free to let me know.

Best regards,
Qiankun Gao

**Comment by Divadi** [comment]
I see - thank you so much for your response! Apologies; I wasn't too familiar with the codebase yet.

**Comment by gqk** [comment]
@Divadi You are welcome. If you encounter any further issues or have additional questions, please don't hesitate to reach out. We're here to assist you in any way we can.
